src/lib/datepicker/datepicker-input.ts

```diff
@@ -43,6 +44,21 @@ export const MD_DATEPICKER_VALIDATORS: any = {
 };
 
 
+/** An event used for datepicker input and change events. */
```

Comment:
Nit: Maybe add something to the comment about why you don't just use the regular event objects from (input) and (change).